context/TripContext.js

```diff
@@ -119,6 +120,23 @@ export function TripProvider({ children }) {
     }
   };
 
+  const checkPermission = async (tripID) => {
+
+    try {
+      const checkPermissionRes = await fetch(`${API_URL}/trip/${tripID}/locked`, {
+        method: 'GET',
+        headers: authHeader,
+      });
+
+      const res = await checkPermissionRes.json()
```

Comment:
maybe use checkStatus from fetch utils?

Reply:
Yeah, it wasn't working at the beginning because I'm stupid and I was trying things.
Later it worked and I forgot to change it back